### ISSUE TITLE
`mv`, `mkdir`, `cp` and `rm` output an empty list for some reason

### ISSUE THREAD

**Issue by webbedspace** [nushell/nushell#6756]
### Describe the bug

For some reason, these commands output empty lists instead of $nothing (which is what `touch` outputs). I really don't understand the reason behind this.

### How to reproduce

```
〉mv 1.txt 1.txt.bak | to nuon
[]
〉cp 1.txt 1.txt.bak | to nuon
[]
〉touch A.txt | to nuon
$nothing
```

### Expected behavior

I expect $nothing to be output for all such commands.
This gets pretty annoying when combined with `each`:
```
〉[1.txt 2.txt] | each {|e| touch $e }
〉[1.txt 2.txt] | each {|e| rm $e }
╭───┬────────────────╮
│ 0 │ [list 0 items] │
│ 1 │ [list 0 items] │
╰───┴────────────────╯
```
In the second example, some completely useless output is given (unlike the first example).

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.69.1                                          |
| branch             |                                                 |
| commit_hash        | 8a5abc7afc28a36908a32f1852127e2a5170b461        |
| build_os           | windows-x86_64                                  |
| build_target       | x86_64-pc-windows-msvc                          |
| rust_version       | rustc 1.64.0 (a55dd71d5 2022-09-19)             |
| rust_channel       | stable-x86_64-pc-windows-msvc                   |
| cargo_version      | cargo 1.64.0 (387270bc7 2022-09-16)             |
| pkg_version        | 0.69.1                                          |
| build_time         | 2022-09-28 02:58:08 +00:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

**Comment by fdncred** [comment]
yes, we need consistency throughout all our commands.

**Comment by rgwood** [comment]
Looks like this is an artifact of how these filesystem commands report success with their `verbose`/`show-created-paths` flags; they return a `list` of strings, :

```
/home/reilly/source/nushell/scratch〉rm -r --verbose 1a
╭───┬────────────────────────────────────────────────╮
│ 0 │ deleted /home/reilly/source/nushell/scratch/1a │
╰───┴────────────────────────────────────────────────╯
〉rm -r --verbose 2a | describe
list<string>
```

By default, they return an empty list. Agreed that this is confusing!

The best way forward is not obvious to me. Are we OK changing the command's return type to return `nothing` by default, and `list<string>` when `--verbose` is set? Or should we consider printing verbose info instead of returning it?

**Comment by webbedspace** [comment]
This is closed by #8014, right?
